docs/developer-guides/getting-started.md

```diff
@@ -0,0 +1,121 @@
+# Getting started
+
+This is a guide for partner chains developers to get started in the project. When you complete it, you should be able to build the project, test locally, and access development environment hosted in EKS.
+
+Prerequisites:
+* `nix` installed
+* `docker` installed
+* `protoc` installed [link](https://google.github.io/proto-lens/installing-protoc.html)
+
+## Nix
+
+The development shells use Nix with flakes enabled. It should provide the most
+important tools (besides `docker`), to work in this project: `cargo` for
+building the project, and `aws` and `kubectl` to interact with the development
+environment.
+
+[Install a recent version of Nix](https://zero-to-nix.com/start/install), and
+use it to enter the project shell, preferably with [`direnv`](#direnv-support),
+but `nix develop` will also work.
+
+In order for nix to access our private repositories, it needs to be aware of a
```

Comment:
I don't think we should have it here. Instead `kubectl` authentication should be moved to our private infra repository and all `kubectl` usage should be there. 

I think that since partner-chains-smart-contracts repo is public now, we should not need a token to have devshell with smart contracts available.

@tgunnoe should have the best knowledge about this, I guess

Reply:
That is true, the sections about authentication are irrelevant for setup now.

Reply:
At least the information should be moved to places where its needed. the .netrc and GITHUB_TOKEN parts could be moved to our private infra, if that will continue to exist. I'm hoping we move mostly everything into the open though